frontend/src/pages/SessionForm/pages/participants.js

```diff
@@ -54,6 +61,50 @@ const Participants = ({ formData }) => {
     fetchRecipients();
   }, [recipientOptions, regionId]);
 
+  // Get Groups.
+  const [groups, setGroups] = useState([]);
```

Comment:
I was thinking of making a new component for groups. However I'm not sure if it would get too complicated as both pages have some differences.

Reply:
what are the differences? always good to leverage components, as that is how react is meant to function (IMO)

Reply:
Property names also a question of how much of it do we make its own component.

Reply:
I spent some time trying to move this to its own component. The crux of the issue is that the activity summary does extra stuff with other entity. I think it makes sense for the time being to keep these separate. I have created a standalone component for groups and recipients on the TR. If we need this again on another page we can use that going forward.